### PR TITLE
#170764517 Ch Travis CI test integration 

### PR DESCRIPTION
**Pull request by Alexander96779**
#### What does this PR do?
This PR integrates Travis CI and add a badge to the ReadME
#### Description of Task to be completed?
- Activate repo on Travis CI
- Integrate Travis CI via `.travis.yml`
- Add a badge to the ReadME
#### How should this be manually tested?
- Visit the Repo, you will see the badge showing build from Travis CI
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#170764517](https://www.pivotaltracker.com/story/show/170764517)

### CODE REVIEW

**Review by Elvis-rugamba — APPROVED** [review]
LGTM

**Review by Skemc — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 .DS_Store
 node_modules
 dist
 package-lock.json
 coverage
 .env
 .vscode
-html-report
\ No newline at end of file
+html-report
```
Comment (ajimae) [inline .gitignore:8]: Why are we having this here, I thought this file is meant to be inside the coverage folder?

Comment (Alexander96779) [reply .gitignore:8]: Let me fix it

**Review by ajimae — APPROVED** [review]

**Pull request merged**